lab2/index.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+    // Peventing Undefined variables
+    $username = "";
```

Comment:
Lijkt me onnodig?

Reply:
Dat komt omdat je de `canLogin()` functie pas moet aanroepen zodra iemand inlogt, namelijk ergens in je `if` statement op lijn 17. Als je daar je check naartoe verplaatst heb je de lege variables niet nodig.